spyder/widgets/tabs.py

```diff
@@ -385,6 +385,16 @@ def keyPressEvent(self, event):
                 handled = True
         if not handled:
             QTabWidget.keyPressEvent(self, event)
+
+    def tab_navigate(self, delta=1):

```

Comment:
@csabella, `Ctrl+Tab` is handled differently in the Editor, because there we show a dialog to select among the current opened files. So I think you need to revert this change.

Reply:
@ccordoba12   It still works correctly in the Editor even with moving this.  Without changing preferences, `Ctrl+Tab` brings up the dialog to select the file in MRU  order and `Ctrl+PgDown` cycles through the files.  But, this code allows those keys to be configurable.

Reply:
O, great! Then this is ready.